doc/ref/reesmat.xml

```diff
@@ -125,14 +125,14 @@ gap> S:=ReesZeroMatrixSemigroup(DihedralGroup(IsPermGroup, 8), mat);
     <Oper Name="ReesZeroMatrixSubsemigroup" Arg="R, I, U, J"/>
     <Returns>A Rees matrix or 0-matrix subsemigroup.</Returns>
     <Description>
-      The arguments of <C>ReesMatrixSubsemigroup</C> should be a Rees matrix
+      The arguments of <Ref Oper="ReesMatrixSubsemigroup"/> should be a Rees matrix
       semigroup <A>R</A>, subsets <A>I</A> and <A>J</A> of the rows and columns
       of <A>R</A>, respectively, and a subsemigroup <A>S</A> of the underlying
```

Comment:
`S` does not occur among the arguments.

Reply:
Indeed, I believe `U` was meant. Will fix